docs/faq.rst

```diff
@@ -124,10 +124,28 @@ Which ML libraries does Fairlearn support?
     `predict()` methods. Also, any classification or regression
     algorithm can be evaluated using our metrics.
 
+Does Fairlearn support multi-class classification?
+    Certain metrics will still work for multi-class classification, e.g.,
+    accuracy score. For others it is more complicated since they rely on
+    binary classes, e.g., true positive rate.
+    There are
+    `ongoing discussions within the community <https://github.com/fairlearn/fairlearn/issues/752>`_
+    to add more extensive support to Fairlearn's assessment capabilities.
+    If you have specific thoughts feel free to add them to the discussion.
+
+Does Fairlearn support multiple and non-binary sensitive features?
+    Fairlearn's assessment capabilities support sensitive features with more
+    than two values as well as multiple sensitive features.
+    Our :ref:`user guide <metrics_with_grouping>` has examples for both of
+    these cases.
+    The mitigation techniques all support mitigation with non-binary and
```

Comment:
While `GridSearch` can theoretically do this, I believe it runs into dimensional problems quite quickly.

Reply:
Yes, but I don't want that level of detail in this high-level FAQ. That belongs in the GridSearch API docs or user guide.